### PR TITLE
Invictus 1.6

### PR DESCRIPTION
**Pull request by TinWiz**
Changes all recorded in changelog

### CODE REVIEW

File: localization/spanish/dummy [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (IhateTrains) [inline localization/spanish/dummy:1]: I think a dummy fine is not needed here, as you have other files in the folder.

File: setup/provinces/te_default.txt [hunk 2]
```diff
@@ -87,16 +98,16 @@
 	trade_goods=""
 }
 #
-5971=
```
Comment (IhateTrains) [inline setup/provinces/te_default.txt:90]: If you're leaving commented out blocks, I suggest adding a comment with an explanation here. Maybe copy the explanation from the changelog.

File: setup/provinces/te_default.txt [hunk 3]
```diff
@@ -87,16 +98,16 @@
...
-{
-	barbarian_power=0
-	civilization_value=0
-	culture="hunnic"
-	province_rank=""
-	religion="eastern_animism"
-	terrain="impassable_terrain"
-	trade_goods=""
-}
+#5971=
```
Comment (IhateTrains) [inline setup/provinces/te_default.txt:101]: If you're leaving commented out blocks, I suggest adding a comment with an explanation here. Maybe copy the explanation from the changelog.

File: events/l6_christian.txt [hunk 4]
```diff
@@ -160,8 +161,8 @@ l6_christian.3 = {
 		NOR = {
 			religion = manichaeism
 			religion = christianity
-			tag = JUD
```
Comment (IhateTrains) [inline events/l6_christian.txt:163]: Why not remove instead of commenting out?

**Review by IhateTrains — APPROVED** [review]

File: events/l6_christian.txt [hunk 5]
```diff
@@ -160,8 +161,8 @@ l6_christian.3 = {
...
-			tag = SMA
+			#tag = JUD
```
Comment (IhateTrains) [inline events/l6_christian.txt:164]: Why not remove instead of commenting out?

File: localization/spanish/dummy (hunk 1 above)
Comment (TinWiz) [reply localization/spanish/dummy]: deleted

File: setup/provinces/te_default.txt (hunk 3 above)
Comment (TinWiz) [reply setup/provinces/te_default.txt]: Removed these, not needed.

File: events/l6_christian.txt (hunk 5 above)
Comment (TinWiz) [reply events/l6_christian.txt:164]: I comment out while testing to track changes in the file itself if I'm not entirely comfortable deleting something. At this point, I can remove.

Thanks!

**Pull request merged**